core/spec.md

```diff
@@ -452,6 +450,12 @@ The definition of each attribute is defined below.
     be based on where the Registry is used. For example, a publicly accessible
     Registry might want to consider using a UUID, while a private Registry
     does not need to be so widely unique.
+  - In cases where an entity's `id` is specified outside of the serialization
+    of the entity (e.g. part of a request URL, or a map key), then its
```

Comment:
```suggestion
    of the entity (e.g. part of a request URL, or a map key), its
```

(Or change to "If an entity's `id` [...]")

Reply:
done